SPECS-EXTENDED/openblas/openblas.spec

```diff
@@ -28,21 +28,25 @@ Patch0:         openblas-0.2.15-system_lapack.patch
 Patch1:         openblas-0.2.5-libname.patch
 # Don't use constructor priorities on too old architectures
 Patch2:         openblas-0.2.15-constructor.patch
+# Fix SBGEMM test to work with INTERFACE64
+Patch3:         openblas-0.3.21-sbgemm-test.patch
 # Supply the proper flags to the test makefile
-Patch3:         openblas-0.3.7-tests.patch
-
-# Fix C++ compatibility (BZ #1820131)
-Patch4:         https://github.com/xianyi/OpenBLAS/commit/ee2e758278b5d82b7242f505ea694f082ef65879.patch
-
+Patch4:         openblas-0.3.11-tests.patch
 # keep this patch to build from a containerized environment
 Patch5:         No-Fortran-Build.patch
 
+BuildRequires: make
 BuildRequires:  gcc
 BuildRequires:  gcc-c++
 BuildRequires:  gcc-gfortran
 BuildRequires:  perl-devel
 BuildRequires:  multilib-rpm-config
 
+# Rblas library is no longer necessary
+%if 0%{?fedora} >= 31 || 0%{?rhel} >= 8
+Obsoletes:      %{name}-Rblas < %{version}-%{release}
+%endif
```

Comment:
Might not be necessary for Fedora/RHEL, but can you confirm that we don't need the Rblas library in Mariner?

Reply:
Discussed offline- no dependencies on this library.